packages/twenty-front/src/modules/ui/layout/page/SubMenuTopBarContainer.tsx

```diff
@@ -28,22 +29,23 @@ export const SubMenuTopBarContainer = ({
   children,
   title,
   actionButton,
-  Icon,
   className,
+  breadcrumbs,
 }: SubMenuTopBarContainerProps) => {
   const isMobile = useIsMobile();
 
   return (
     <StyledContainer isMobile={isMobile} className={className}>
-      <PageHeader
-        title={title}
-        Icon={Icon}
-        width={OBJECT_SETTINGS_WIDTH + 4 * 8}
-      >
-        {actionButton}
-      </PageHeader>
+      <PageHeader title={breadcrumbs}>{actionButton}</PageHeader>
       <PageBody>
         <InformationBannerWrapper />
+        <PageTitle>
```

Comment:
Couldn't we just force passing a text as a props and harmonize everything into H1 ?